app/views/layouts/base.html.erb

```diff
@@ -117,7 +117,7 @@ See COPYRIGHT and LICENSE files for more details.
       <% if show_onboarding_modal? %>
         <section data-augmented-model-wrapper
                  data-modal-initialize-now="true"
-                 data-modal-class-name="onboarding-modal op-modal_autoheight -highlight">
+                 data-modal-class-name="onboarding-modal spot-modal_autoheight -highlight">
```

Comment:
The class ` spot-modal_autoheight` does not exist. I assume it should be either removed completely, or be just `spot-modal`?

Reply:
as far as I understand it, this one receives the `spot-modal` through `data-modal-initialize-now`. otherwise it would now have worked. I try to find the usage of that one.

Reply:
well, the `spot-modal` is within the partial rendered in there.